packages/discovery-provider/plugins/pedalboard/apps/solana-relay/src/routes/relay/relay.ts

```diff
@@ -105,23 +88,23 @@ export const relay = async (
     const feePayerKey = decompiled.payerKey
     const feePayerKeyPair = getFeePayerKeyPair(feePayerKey)
 
-    try {
-      await assertRelayAllowedInstructions(decompiled.instructions, {
-        user: res.locals.signerUser,
-        feePayer: feePayerKey.toBase58()
-      })
-    } catch (e) {
-      if (e instanceof InvalidRelayInstructionError) {
-        throw new BadRequestError('Invalid relay instructions', { cause: e })
-      } else {
-        throw e
-      }
-    }
-
     if (feePayerKeyPair) {
       res.locals.logger.info(
         `Signing with fee payer '${feePayerKey.toBase58()}'`
       )
+      try {
```

Comment:
this seems reasonable.. but can you remind me, what's an example of a tx that would get relayed that we're not signing/paying for? would this change allow any arbitrary tx to use our RPC and isn't that a bit unsafe?

Reply:
this could be the buy audio flow, or the withdrawal swap prior to this change (i made this change before i switched that out)

The win here is we can use our RPCs w/o risk of abuse of our fee payer, and get our retry logic etc. You're right though that we might want some further protection... will think about this